app/build.gradle

```diff
@@ -37,6 +37,7 @@ dependencies {
     implementation 'org.springframework.boot:spring-boot-starter-security'
     implementation 'org.springframework.boot:spring-boot-starter-actuator'
     implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
+    implementation 'org.springframework.boot:spring-boot-starter-amqp:2.7.4'
```

Comment:
Can we read the version from settings file? This is also used in service-data-access.

Reply:
ok, I removed the version